webapp/src/js/controllers/reports.js

```diff
@@ -314,6 +315,9 @@ angular
           })
           .catch(function(err) {
             $log.error('Error verifying message', err);
+          })
+          .finally(() => {
+            $scope.loadingContent = false;
```

Comment:
At this point we know the update was successful but the UI won't be updated till the change event fires. This means we'll see the old state, then the spinner, then the old state, then the new state which is all a bit confusing. Soon admins won't have change events at all so the spinner will stop but the report will still look like it's wrong.

I think we should add a new line here which manually sets `$scope.selected[0].verified = valid`. That way the UI will be in sync with the DB.